processing/src/main/java/org/apache/druid/query/aggregation/ExpressionLambdaAggregatorFactory.java

```diff
@@ -121,6 +124,7 @@ public ExpressionLambdaAggregatorFactory(
 
     this.initialValueExpressionString = initialValue;
     this.initialCombineValueExpressionString = initialCombineValue == null ? initialValue : initialCombineValue;
+    this.initiallyNull = initiallyNull == null ? NullHandling.sqlCompatible() : initiallyNull;
```

Comment:
This could be named as `useNullInitially` or `isInitiallyNull` to make it more clear in code. As above, I would also be fine with some other better name.

Reply:
renamed to `isNullUnlessAggregated` to more clearly indicate that it is a boolean and hopefully indicate its main role in determining aggregator behavior. `initiallyNull` seemed confusing alongside `initialValue`.